Bearded.Utilities.Testing/Core/MaybeAssertions.cs

```diff
@@ -0,0 +1,52 @@
+using FluentAssertions;
+using FluentAssertions.Execution;
+
+namespace Bearded.Utilities.Testing
+{
+    public sealed class MaybeAssertions<T>
+    {
+        private readonly Maybe<T> subject;
+
+        public MaybeAssertions(Maybe<T> instance) => subject = instance;
+        
+        [CustomAssertion]
+        public void BeJust(T value, string because = "", params object[] becauseArgs)
+        {
+            BeJust().Which.Should().BeEquivalentTo(value, because, becauseArgs);
```

Comment:
Why equivalent? I'd expect this to test equality.

Reply:
I thought this was recommended practice, but looking at the documentation again, it seems like they actually discourage it. Changed.